test/movie_bloc_test.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:movies/src/blocs/movie_bloc.dart';
+import 'package:movies/src/models/movie.dart';
+import 'package:movies/src/models/movie_result.dart';
+import 'package:movies/src/resources/movie_repository.dart';
+import 'package:movies/src/ui/home_page.dart';
+
+import 'movie_bloc_test.mocks.dart';
+
+@GenerateMocks([
+  MovieRepository,
+])
+void main() {
+  late Movie movieEmpty;
+  late MockMovieRepository repository;
+  late MovieBloc bloc;
+  late MovieResult movieResult;
+  late Movie movieFull;
+
+  Movie _movieList(int pageParameter, int totalPagesParameter,
+      int totalResultsParameter, List<MovieResult> resultsParameter) {
+    return Movie(
+      page: pageParameter,
+      totalPages: totalPagesParameter,
+      totalResults: totalResultsParameter,
+      results: resultsParameter,
+    );
+  }
+
+  setUp(() {
+    movieEmpty = _movieList(1, 5, 3, []);
```

Comment:
You can change this variable name to more descriptive, because the structure that you're using it is not for a movie

Reply:
done